packages/components/nodes/llmcache/LLMMomentoCache/LLMMomentoCache.ts

```diff
@@ -0,0 +1,75 @@
+import {
+    getBaseClasses,
+    getCredentialData,
+    getCredentialParam,
+    ICommonObject,
+    INode,
+    INodeData,
+    INodeOutputsValue,
+    INodeParams
+} from '../../../src'
+import { MomentoCache } from 'langchain/cache/momento'
+import { CacheClient, Configurations, CredentialProvider } from '@gomomento/sdk'
+
+class LLMMomentoCache implements INode {
+    label: string
+    name: string
+    version: number
+    description: string
+    type: string
+    icon: string
+    category: string
+    baseClasses: string[]
+    inputs: INodeParams[]
+    outputs: INodeOutputsValue[]
+    credential: INodeParams
+
+    constructor() {
+        this.label = 'Momento Cache'
+        this.name = 'momentoCache'
+        this.version = 1.0
+        this.type = 'LLMCache'
+        this.icon = 'momento.png'
+        this.category = 'LLM Cache'
+        this.baseClasses = [this.type, 'LLMCacheBase']
+        this.credential = {
+            label: 'Connect Credential',
+            name: 'credential',
+            type: 'credential',
+            optional: true,
+            credentialNames: ['momentoCacheApi']
+        }
+        this.inputs = []
+        this.outputs = [
+            {
+                label: 'LLM Cache',
+                name: 'cache',
+                baseClasses: [this.type, ...getBaseClasses(MomentoCache)]
+            }
+        ]
+    }
+
+    async init(nodeData: INodeData, _: string, options: ICommonObject): Promise<any> {
+        const credentialData = await getCredentialData(nodeData.credential ?? '', options)
+        const apiKey = getCredentialParam('momentoApiKey', credentialData, nodeData)
+        const cacheName = getCredentialParam('momentoCache', credentialData, nodeData)
+        const endPoint = getCredentialParam('momentoEndpoint', credentialData, nodeData)
```

Comment:
is this being used?

Reply:
miss at my end will clean up the unsed vars.

Reply:
Done! Removed the unused vars.